t3/common.py

```diff
@@ -47,3 +51,34 @@ def delete_root_rmg_log(project_directory: str) -> None:
     rmg_log_path = os.path.join(project_directory, 'RMG.log')
     if os.path.isfile(rmg_log_path):
         os.remove(rmg_log_path)
+
+
+def get_species_by_label(label: str,
```

Comment:
I see you already did this here, please squash/organize the commits so we won't have addition+deletion in the same PR.